plate_simulation/__init__.py

```diff
@@ -12,7 +12,7 @@
 from pathlib import Path
 
 
-__version__ = "0.1.0-beta.3"
+__version__ = "0.2.0-alpha.1"
```

Comment:
We've never officially released it, so stick to 0.1.0-beta.3

Reply:
done.